src/broadcast_service.rs

```diff
@@ -227,7 +227,10 @@ impl BroadcastService {
             if exit_signal.load(Ordering::Relaxed) {
                 return BroadcastServiceReturnType::ExitSignal;
             }
-            let mut broadcast_table = cluster_info.read().unwrap().sorted_tvu_peers(&bank);
+            let mut broadcast_table = cluster_info
+                .read()
+                .unwrap()
+                .sorted_tvu_peers(&bank.get_stakes());
```

Comment:
This function, `get_stakes()` is misleading. How about renaming that to `staked_validators()` and documenting it with something like: a staker with non-zero balance has delegated validation do the `node_id` returned in the `(node_id, stake)` pair.

Reply:
@garious, Yeah I was thinking about this on the drive home. I think it should just be "staked_nodes".
Can you help me understand the second part of your comment? A staker with no balance has delegated validation?

Reply:
Not "no balance". "non-zero balance".  Technically the Vote program can be referencing a staker that has emptied its account. We need to make sure to drop those.